include/bluetooth/gap.h

```diff
@@ -69,6 +69,30 @@ extern "C" {
 #define BT_GAP_INIT_CONN_INT_MIN                0x0018  /* 30 ms    */
 #define BT_GAP_INIT_CONN_INT_MAX                0x0028  /* 50 ms    */
 
+/* LE PHY types */
+#define BT_GAP_LE_PHY_1M                        0x01 /* LE 1M PHY */
+#define BT_GAP_LE_PHY_2M                        0x02 /* LE 2M PHY */
+#define BT_GAP_LE_PHY_LONG_RANGE                0x03 /* LE Coded PHY */
+
+/* Advertising PDU types */
+#define BT_GAP_ADV_TYPE_ADV_IND                 0x00
+#define BT_GAP_ADV_TYPE_ADV_DIRECT_IND          0x01
+#define BT_GAP_ADV_TYPE_ADV_SCAN_IND            0x02
+#define BT_GAP_ADV_TYPE_ADV_NONCONN_IND         0x03
+#define BT_GAP_ADV_TYPE_SCAN_RSP                0x04
+
+/* Advertising PDU properties */
+#define BT_GAP_ADV_PROP_CONNECTABLE             BIT(0)
+#define BT_GAP_ADV_PROP_SCANNABLE               BIT(1)
+#define BT_GAP_ADV_PROP_DIRECTED                BIT(2)
+#define BT_GAP_ADV_PROP_SCAN_RESPONSE           BIT(3)
+#define BT_GAP_ADV_PROP_LEGACY                  BIT(4)
```

Comment:
I would like to use a different name than LEGACY here. Legacy is not necessarily the right word. It would imply that the advertising is old and should not be used. But that is not the case. If you don't have to send much data, it is more effective to send it using "legacy" advertising.

Best I could come up with so far is "Regular" advertising or "Non_extended".
Alternatively I would have like to flip the meaning, so that the bit is set to indicate Extended, as opposed to indicating "Legacy."

Reply:
> Alternatively I would have like to flip the meaning, so that the bit is set to indicate Extended, as opposed to indicating "Legacy."

Of the options you presented that sounds like the best one to me.